rollup/internal/controller/relayer/l1_relayer.go

```diff
@@ -219,3 +231,29 @@ func (r *Layer1Relayer) StopSenders() {
 		r.gasOracleSender.Stop()
 	}
 }
+
+func (r *Layer1Relayer) shouldUpdateGasOracle(baseFee uint64, blobBaseFee uint64, isCurie bool) bool {
+	// Right after restarting.
+	if r.lastBaseFee == 0 {
+		return true
+	}
+
+	expectedBaseFeeDelta := r.lastBaseFee*r.gasPriceDiff/gasPriceDiffPrecision + 1
+	if baseFee >= r.minGasPrice && (baseFee >= r.lastBaseFee+expectedBaseFeeDelta || baseFee+expectedBaseFeeDelta <= r.lastBaseFee) {
+		return true
+	}
+
+	// Omitting blob base fee checks before Curie.
+	if !isCurie {
+		return false
+	}
```

Comment:
```suggestion
	}
	
	if r.lastBlobBaseFee == 0 {
		return true
	}
```

Reply:
added in [23f46af](https://github.com/scroll-tech/scroll/pull/1344/commits/23f46afcb55dae3d8b317dec27d537b9fbd61f33).